src/chrome/content/rules/snapraid.it.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="snapraid.it">
+	<target host="snapraid.it" />
```

Comment:
This host does not exist, but `beta.snapraid.it` does (mismatching cert)

Reply:
Do I have to add domains withh mismatching certs? Or just delete snapraid.it?